### PR TITLE
Instructions for creating a conda enviroment with python3.8.8

### PR DESCRIPTION
**Pull request by melhemr**
PR will add instructions for creating a conda environment to ensure python version is 3.8.8

### CODE REVIEW

**Comment by melhemr** [comment]
@AbdulkadirA 

The only line I wasn't entirely sure how to deal with this line since I am using a mac: 

### Prepare environment for PowerShell (Windows 10 or 11)
Assuming conda enviroment with Python version 3.8.8 has been created

```shell
**& env/Scripts/Activate.ps1**
python -m pip install --upgrade pip
poetry install
```

In these instructions, I've outlined it so the users create a conda enviroment, so I'm not sure where or how the ```& env/Scripts/Activate.ps1``` should be implemented for Windows users. Can you let me know?

**Comment by ashishsingh18** [comment]
> 
@melhemr In my conda env, I usually just do:
`python -m pip install -U -e .`
`poetry install`

**Comment by ashishsingh18** [comment]
It might be clearer if we create 2 separate sections for installation: one using conda and the other using virtual env.

File: README.md [hunk 1]
```diff
@@ -25,31 +25,53 @@ User documentation is available at
...
+After activating the environment, verify the version with
+
+```shell
+python --version # should be 3.8.8
 ```
 
 ### Prepare environment in Linux (CUBIC)
-Assuming current working directory is `NiBAx` and containing the source code
-cloned from https://github.com/CBICA/NiBAx.git.
+Assuming conda enviroment with Python version 3.8.8 has been created
```
Comment (AbdulkadirA) [inline README.md:64]:
```suggestion
Assuming conda enviroment with Python version 3.8.8 or newer has been created
```

File: README.md [hunk 2]
```diff
@@ -25,31 +25,53 @@ User documentation is available at
...
 ```shell
-python -m venv .env
-.env/bin/activate
 python -m pip install --upgrade pip
 ```
 
 ### Prepare environment for PowerShell (Windows 10 or 11)
-Assuming current working directory is `NiBAx` and containing the source code
-cloned from https://github.com/CBICA/NiBAx.git.
+Assuming conda enviroment with Python version 3.8.8 has been created
```
Comment (AbdulkadirA) [inline README.md:71]:
```suggestion
Assuming conda enviroment with Python version 3.8.8 or newer has been created
```

File: README.md [hunk 3]
```diff
@@ -25,31 +25,53 @@ User documentation is available at
...
+Assuming current working directory is `NiBAx` and containing the source code
+cloned from https://github.com/CBICA/NiBAx.git.
+
+Ensure Anaconda is installed. Follow instructions for user's operating system [here](https://docs.anaconda.com/anaconda/install/index.html). After Anaconda has been installed, be sure to exit and reopen any 
+command line windows to use `conda` command
+
+To create the enviroment:
+
+```shell
+conda create -n env python=3.8.8
```
Comment (AbdulkadirA) [inline README.md:42]: Calling the environment `env` may be misleading. I think it is much better to call the environment `NiBAx`.

File: README.md [hunk 4]
```diff
@@ -25,31 +25,53 @@ User documentation is available at
...
 
 ```shell
-python --version # should be 3.8.8 or newer
+conda deactivate
+```
+
+After activating the environment, verify the version with
+
+```shell
+python --version # should be 3.8.8
```
Comment (AbdulkadirA) [inline README.md:60]:
```suggestion
python --version # should be 3.8.8 or newer
```

**Review by AbdulkadirA — CHANGES_REQUESTED** [review]
@melhemr Thanks. I made some suggestions.

File: README.md [hunk 5]
```diff
@@ -25,31 +25,53 @@ User documentation is available at
 
 
 ## Setup for development
-Install Python version 3.8.8 or newer.
+Install Python version 3.8.8.
```
Comment (AbdulkadirA) [inline README.md:28]:
```suggestion
Install Python version 3.8.8 or newer.
```

**Review by AbdulkadirA — COMMENTED** [review]
And one more comment ..

**Review by AbdulkadirA — CHANGES_REQUESTED** [review]
@melhemr Thanks. Please leave the sections on setting up the PIP venv for Windows and Linux as they were and only add a section for conda. You could add the indication of PIP to the exiting section headers, though.

**Comment by melhemr** [comment]
> @melhemr Thanks. Please leave the sections on setting up the PIP venv for Windows and Linux as they were and only add a section for conda. You could add the indication of PIP to the exiting section headers, though.

Can you clarify what you mean? I did leave the PIP section alone, and added conda sections as we discussed. Windows for example:

```
### Prepare environment for PowerShell (Windows 10 or 11)
Assuming current working directory is `NiBAx` and containing the source code
cloned from https://github.com/CBICA/NiBAx.git.

Virtual environment
```shell
python -m venv .env
& .env/Scripts/Activate.ps1
python -m pip install --upgrade pip
poetry install
```
```
Conda environment (assuming conda environment with Python version 3.8.8 or newer has been created)
```shell
conda activate NiBAx
python -m pip install --upgrade pip
poetry install
```

File: README.md [hunk 6]
```diff
@@ -25,14 +25,34 @@ User documentation is available at
 
 
 ## Setup for development
-Install Python version 3.8.8 or newer.
-The exact procedure depends on the operating system and configuration.
-Verify the version with
+Install Python version 3.8.8 or newer. The exact procedure depends on the operating system and configuration. Verify the version with
```
Comment (AbdulkadirA) [inline README.md:28]:
```suggestion
Install Python version 3.8.8 or newer.
The exact procedure depends on the operating system and configuration.
Verify the version with
```

File: README.md [hunk 7]
```diff
@@ -25,14 +25,34 @@ User documentation is available at
...
 
 ```shell
 python --version # should be 3.8.8 or newer
 ```
 
+If python environment is not 3.8.8 or newer, use the following instructions to prepare a conda environment with the correct version.
```
Comment (AbdulkadirA) [inline README.md:34]:
```suggestion
```

File: README.md [hunk 8]
```diff
@@ -25,14 +25,34 @@ User documentation is available at
...
+
+### Prepare conda environment
+Assuming current working directory is `NiBAx` and containing the source code
+cloned from https://github.com/CBICA/NiBAx.git.
+
+Ensure Anaconda is installed. Follow instructions for user's operating system [here](https://docs.anaconda.com/anaconda/install/index.html). After Anaconda has been installed, be sure to exit and reopen any 
+command line windows to use `conda` command
+
+To create the enviroment:
```
Comment (AbdulkadirA) [inline README.md:43]:
```suggestion
```

File: README.md [hunk 9]
```diff
@@ -25,14 +25,34 @@ User documentation is available at
...
+
+```shell
+conda create -n NiBAx python=3.8.8  
+```
+
+Activate conda environment (applicable to PowerShell and Linux)
+```shell
```
Comment (AbdulkadirA) [inline README.md:50]:
```suggestion
```

File: README.md [hunk 10]
```diff
@@ -54,6 +74,7 @@ python -m pip install --upgrade pip
 poetry install
 ```
 
+
 ### Install the [NiBAx] software
 To install the [NiBAx], install it in a virtual environment. Depending on the
 desired version, use one of the following commands to install it.
```
Comment (AbdulkadirA) [inline README.md:80]:
```suggestion
To install the [NiBAx], install it in a virtual or conda environment.
Depending on the desired version, use one of the following
commands to install it.
```

**Review by AbdulkadirA — CHANGES_REQUESTED** [review]
@melhemr I propose a couple of changes to further simplify.

**Comment by melhemr** [comment]
@AbdulkadirA the changes look good. The only final concern I had is about Ashish's comment above. It seems that using a conda environment with a Windows OS still requires using the command ```poetry install```. I don't know if that's 100% necessary, but if it is, how should we include that in the conda environment section?

File: README.md [hunk 11]
```diff
@@ -54,9 +70,11 @@ python -m pip install --upgrade pip
 poetry install
```
Comment (AbdulkadirA) [inline README.md:70]:
```suggestion
```

**Review by AbdulkadirA — CHANGES_REQUESTED** [review]
Poetry is generally not required, only to package the toolbox.

**Review by AbdulkadirA — APPROVED** [review]

**Pull request merged**